spiceai/README.md

```diff
@@ -0,0 +1,125 @@
+## Spice Quickstart Tutorial using the Spice.ai Cloud Platform
+
+The Spice.ai Cloud Platform has many datasets that can be used within Spice.  A valid login for the Spice.ai Cloud Platform is required to access the datasets.  Before beginning this quickstart, [link your GitHub account to Spice.ai](https://spice.ai/login) to get access to the platform.
+
+**Step 1.** Log in to Spice.ai to store your access key and token in the spice runtime configuration.
+
+```bash
+spice login
+```
+A browser window will open displaying a code that will appear in the terminal.  Select Approve if the authorization codes match.
+
+![Screenshot](./device_login.png)
+
+There will be a confirmation in the terminal that login was successful:
+
+```bash
+Successfully logged in to Spice.ai as your_user (your_email@email.com)
+Using app your_user/your_app
+```
+
+**Step 2.** Initialize a Spice project and start the runtime:
+
+```bash
+spice init spiceai-demo
+```
+
+```bash
+cd spiceai-demo
+spice run
+```
+
+**Step 3.** Configure the dataset to connect to Spice.ai:
+
+```bash
+spice dataset configure
+```
+
+Enter the name of the dataset:
+
+```bash
+dataset name: (spiceai-demo)  eth_recent_blocks
+```
+
+Enter the description of the dataset:
+
+```bash
+description: ethereum recent block data
+```
+
+Specify the location of the dataset:
+
+```bash
+from: spiceai/eth.recent_blocks
+```
+
+Select "n" when prompted whether to locally accelerate the dataset:
+
+```bash
+Locally accelerate (y/n)? n
+```
+
+The CLI will confirm the dataset has been configured with the following output:
+
+```bash
+Saved datasets/eth_recent_blocks/dataset.yaml
+```
+
+Thhe content of dataset.yaml is the following:
```

Comment:
```suggestion
The content of dataset.yaml is the following:
```

Reply:
Also might be worth having the command `cat datasets/eth_recent_blocks/dataset.yaml` so a user can check it.

Reply:
added.  Thanks for the feedback.